src/styles/blocks/carousel.scss

```diff
@@ -0,0 +1,110 @@
+.carousel {
+  display: flex;
+  gap: 20px;
+  overflow-x: scroll;
+  scroll-behavior: smooth;
+  
+  &::-webkit-scrollbar {
+    background-color: transparent;
+    height: 0;
+  }
+
+  &__card {
+    box-sizing: border-box;
+    display: block;
+    margin: 0 -20px 25px;
+    transition: transform 300ms;
+    text-decoration: none;
+    color: $main-color;
+
+    &:hover {
+      transform: scale(1.05);
+    }
+
+    @include Tablet {
+      margin: 0 0 50px;
+    }
+    @include Desktop {
+      margin: 0;
+    }
+
+    &--image-container {
```

Comment:
The class name should be '__image-container' instead of '&--image-container' following the BEM naming convention.